bootstrap/lib/cache.js

```diff
@@ -23,6 +32,10 @@ const envOptions = () => ({
     ...DEFAULT_IGNORED_KEYS,
     ...(process.env.CACHE_IGNORED_KEYS || '').split(',').filter((k) => k), // no empty keys
   ],
+  // Request coalescing
+  requestCoalescingEnabled:
```

Comment:
Would `remoteCaching` be a more simple terminology? `requestCoalescingEnabled` does not really allude distributed caching support

Reply:
As per our offline discussion, request coalescing is a separate enhancement from the remote caching support.